Project.toml

```diff
@@ -17,6 +17,8 @@ ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
 
 [compat]
 ADNLPModels = "0.3.1"
+ExpressionTreeForge = "0.1.0"
+julia = "^1.6.0"
 JuMP = "0.21.5"
```

Comment:
What about JuMP?

Reply:
In `get_expr_tree` function, that supports `MatHOptNLPModel`, I use `JuMP` methodsµ.
There is no alternative.